### ISSUE TITLE
Reward sequencing

### ISSUE THREAD

**Issue by LemADEC** [kristi71111/BaconParty#1]
As of 1.0.4, the sequencing of the vote party is a bit inconsistent:
1- execute player specific commands
2- send player specific message
3- send global message
4- execute global commands

It would make more sense to send the message before executing commands, and generally address global before player specifics. Eventually, there could be global closure global message to say something like "All presents have been dispatched!".

Also, in reference to the original plugin, it would nice to have a small delay, like 10 s before actually executing the rewards so players get time to free some space in their inventory (or whatever preparation they need for the party).

In other words, sequencing could be something like:
1- send global initial message
2- execute global initial commands
3- wait 10 seconds
4- send player specific message
5- execute player specific commands
6- send global closure message
7- execute global closure commands

**Comment by kristi71111** [comment]
Sorry for long delay on response.
[This commit](https://github.com/kristi71111/BaconParty/commit/aaab377ca00b764bdcd85a859f4db6a6fd5a5d38) makes it so that it goes in this order:
1- send global message
2- execute global commands
3- wait 5 seconds
4- send player specific message
5- execute player specific commands
